include/envoy/http/conn_pool.h

```diff
@@ -77,6 +77,12 @@ class Instance : public Envoy::ConnectionPool::Instance, public Event::DeferredD
    */
   virtual Cancellable* newStream(Http::ResponseDecoder& response_decoder,
                                  Callbacks& callbacks) PURE;
+
+  /**
+   * Returns a user-friendly protocol descipton for logging.
```

Comment:
Is protocol description the right thing here? if longer term we want to add cell vs Wi-Fi, should that be reflected? If that the case, should this be named `description`?

Reply:
My thought is we'd have separate strings for each, and combine  in the pool, but we can rename if we end up going the other way.